pkg/arvo/sys/lull.hoon

```diff
@@ -1401,6 +1401,9 @@
         ::  gall scry endpoint
         ::
         [%scry ~]
+        ::  internal name endpoint, returns ship @p
+        ::
+        [%name ~]
```

Comment:
I would consider renaming this (and the `/~/name` path) to say `host` instead. There may or may not be a world in which you can authenticate as yourself on others' ships, and this naming would be ambiguous.

Reply:
Don't like `host`. My idea was for the endpoint to answer the question "what is the name of this ship?", whereas if the question is "what is the host of this ship?" my mind immediately goes to stuff like hosting providers which are unrelated. What do you think about `/~/our`?

Reply:
As discussed out of band, `/~/name` is probably right. Might be worth updating the comment to explicitly say it's the name of the ship the requester is authenticated as though, to avoid confusion about this in the future.